### DISCUSSION TITLE
klipperScreen Console flashing

### DISCUSSION THREAD

**Discussion by dadoblu2000** [KlipperScreen/KlipperScreen#1347] [Q&A]
[Consoles] flashed, I would like to start by saying that before the installation of klipperscreen the console was displayed when the raspberry was started up to the access request, currently after the installation of klipperscreen when the raspberri pi 3b+ was started the data began to scroll in the display until reaching the login request but with monitor flashing, I tried to view the log file but it is not visible in mainsail, I created them with command lines, I try to attach them 
[KlipperScreen_journalctl.log](https://github.com/KlipperScreen/KlipperScreen/files/15141678/KlipperScreen_journalctl.log)
[KlipperScreen_systemctl.log](https://github.com/KlipperScreen/KlipperScreen/files/15141688/KlipperScreen_systemctl.log)
[KlipperScreen_Xorg.log](https://github.com/KlipperScreen/KlipperScreen/files/15141696/KlipperScreen_Xorg.log)

the command 
cp -n /tmp/KlipperScreen.log ~/printer_data/logs/KlipperScreen.log

returns 
cp: cannot stat '/tmp/KlipperScreen.log': No such file or directory

**Comment by T3rbium** [comment]
I think there is an error in the install script. I had the same problem and resolved it by manually changing the script.

remove the . in line 5:
KSENV="${KLIPPERSCREEN_VENV:-${HOME}/.KlipperScreen-env}"

to:
KSENV="${KLIPPERSCREEN_VENV:-${HOME}/KlipperScreen-env}"

**Reply by dadoblu2000** [reply]
[Desktop Entry]
Name=KlipperScreen
GenericName=Touch screen GUI for Klipper via Moonraker
Icon=KlipperScreen
**Exec=sh -c "$HOME/.KlipperScreen-env/bin/python $HOME/KlipperScreen/screen.py"**
Terminal=false
Type=Application
Categories=Graphics;3DGraphics;Engineering;
Keywords=3D;Printing
StartupNotify=false
StartupWMClass=klipper-screen

**Comment by alfrix** [comment]
`ModuleNotFoundError: No module named 'gi'` shows that the dependencies are not correctly installed

@T3rbium the env folder is supposed to have the dot at the front, it always had it

**Reply by dadoblu2000** [reply]
I noticed that the display indicates an anomaly with undervoltage written, is it possible that the raspberry and monitor are not receiving the correct power supply and for this reason some protection is taking over which inhibits the correct installation of the dependencies? I am currently powering with 5v, on the mainselle console it shows me undervoltage 4.65v I will change the power supply with an original 5.1v one and I will try the installation again, I will update the result

**Comment by dadoblu2000** [comment]
ok, I changed the power supply, now I use an original 5.1 volt one, I carried out a new installation and everything seems to work now, the only problem is when I have kliperscreen turned on on the raspberry display I don't have access to the ssh console and the mainsail web page, for In order to have access I have to remove power from the display, at that point I have full access from the PC, any ideas on what the problem is?

**Reply by alfrix** [reply]
just some ideas based previous users reports: some screens act as a shield and block the wifi signal if they are too close to the raspberry, wrongly installed or defective display cables can cause the pi to not start or behave strange
